.circleci/config.yml

```diff
@@ -210,87 +251,96 @@ commands:
 
 jobs:
   notify_of_approval:
-    <<: *app_containers
+    executor: cloud-platform-executor
     steps:
       - *notify_slack_of_approval
 
-  test:
-    <<: *app_containers
+  setup_test_environment:
+    executor: test-executor
     steps:
-      - prepare_tests
-
-      - run:
-          name: rubocop
-          command: bundle exec rubocop
-
-      - run:
-          name: run tests
-          command: COVERAGE=1 bundle exec rspec
-
-      - *notify_slack_on_failure
+      - checkout
+      - setup_remote_docker
+      - build-base
+      - seed-database
 
   api_docs:
-    <<: *app_containers
+    executor: test-executor
+    parallelism: 8
     steps:
-      - prepare_tests
-
-      - run:
-          name: generate swagger documentation
-          command: bundle exec rspec spec/swagger spec/requests --format Rswag::Specs::SwaggerFormatter --order defined
-
+      - checkout
+      - build-base
+      - *api_docs
       - persist_to_workspace:
           root: .
           paths:
             - swagger/v1/swagger.yaml
             - swagger/v2/swagger.yaml
-
       - *notify_slack_on_failure
 
+  rspec_tests:
+    executor: test-executor
+    parallelism: 8
+    steps:
+      - checkout
+      - build-base
+      - *wait-for-db
+      - *create-db
+      - *attach-tmp-workspace
+      - *rspec
+
+  rubocop_tests:
```

Comment:
```suggestion
  linters:
```

Reply:
I suspect we'll want to extend these to include some security checks with brakeman, etc

Reply:
Yeah! Good point.